### ISSUE TITLE
extension and register it

### ISSUE THREAD

**Issue by 100n1k** [onsip/SIP.js#1073]
Hello,

My name is Breno, and I'm from Brazil. I'm reaching out because I could use some help with the basics of setting up a SIP agent. I'm trying to log in with an extension and register it.

I attempted to do this through the Git repository, but it seems to be outdated, resulting in various errors. Due to my limited experience with the sip.js API, I'm having difficulty resolving these issues. Therefore, I'm seeking a starting point to make progress.

If someone could kindly create a simple class for me to register the extension, I would be very grateful. With that, I can try to move forward and develop from there.

Basically, I'm going to try to create an app where I register an extension. Then, I provide a list of phone numbers for the extension to call each number in the list and play an audio that I will make available for the extension.
If I can accomplish this, it could change the life of my family and mine.

Thank you very much for your help.

Best regards, Breno

**Comment by luanyata** [comment]
Hello Breno, welcome. So, can you take a look at SimpleUser? I think it's the easiest way to implement it. [Link: https://sipjs.com/guides/simple-user/]
